src/hearings/containers/request-hearing/hearing-check-answers/hearing-check-answers.component.spec.ts

```diff
@@ -0,0 +1,51 @@
+import {ComponentFixture, TestBed} from '@angular/core/testing';
+import {provideMockStore} from '@ngrx/store/testing';
+import {of} from 'rxjs';
+import {ACTION} from '../../../models/hearings.enum';
+import {HearingsService} from '../../../services/hearings.service';
+import {HearingCheckAnswersComponent} from './hearing-check-answers.component';
+
+describe('CheckAnswersComponent', () => {
+  let component: HearingCheckAnswersComponent;
+  let fixture: ComponentFixture<HearingCheckAnswersComponent>;
+  const mockedHttpClient = jasmine.createSpyObj('HttpClient', ['get', 'post']);
+  const hearingsService = new HearingsService(mockedHttpClient);
+  hearingsService.navigateAction$ = of(ACTION.CONTINUE);
+
+  const initialState = {
+    hearings: {
+      hearingList: {
+        caseHearingMainModel: [
+          {
```

Comment:
indentation

Reply:
can't see any indention issue here, could you double check.